core-plugins/src/main/java/co/cask/hydrator/plugin/batch/source/FileSourceConfig.java

```diff
@@ -213,4 +219,8 @@ protected Schema getSchema() {
   }
 
   protected abstract String getPath();
+
+  boolean shouldCopyHeader() {
```

Comment:
why do we need this method?

Reply:
using it to make it return a boolean instead of a Boolean to avoid warnings about possible nulls. The variable is annotated as nullable to tell cdap that it's optional even though we always set it to a value.